crates/subspace-wasm-tools/src/lib.rs

```diff
@@ -47,40 +46,51 @@ pub fn create_runtime_bundle_inclusion_file(
                 out_components.next();
             }
 
-            if let Component::Normal(profile) = out_components
-                .next()
-                .expect("OUT_DIR has CARGO_TARGET_DIR as prefix")
-            {
-                profile
-                    .to_str()
-                    .expect("Profile is always a utf-8 string")
-                    .to_string()
-            } else {
-                unreachable!("OUT_DIR has CARGO_TARGET_DIR as prefix")
-            }
+            let mut path = target_dir;
+
+            let profile = loop {
+                if let Component::Normal(maybe_profile) = out_components
```

Comment:
`maybe_foo` is more common for an Option type in our codebase, but it's not here.

Reply:
I know, but that variable doesn't necessarily contain a profile, better name is welcome

Reply:
Changed to `profile_or_target`